fip-0026.md

```diff
@@ -339,251 +409,281 @@ This sets the marketplace configurations.
-|Invalid listing fee|`listingfee` is not between 0-10|400| | |"Listing fee can only be between 0 and 10"|
+|Invalid marketplace|Supplied `marketplace` is not found in marketplace config table `mrkplconfig`|400|"marketplace"|Value sent in, e.g. "notamarketplace"|"Marketplace not found"|
+|Invalid listing fee|`fee` is not between 0-10|400|"fee"|Value sent in, e.g. "20" |"Listing fee can only be between 0 and 10"|
+|Invalid fee value|max_fee format is not valid|400|"max_fee"|Value sent in, e.g. "-100"|"Invalid fee value"|
+|Fee exceeds maximum|Actual fee is greater than supplied max_fee|400|"max_fee"|Value sent in, e.g. "1000000000"|"Fee exceeds supplied maximum"|
+|Insufficient funds to cover fee|Account does not have enough funds to cover fee|400|"max_fee"|Value sent in, e.g. "1000000000"|"Insufficient funds to cover fee"|
+|Invalid TPID|tpid format is not valid|400|"tpid"|Value sent in, e.g. "notvalidfioaddress"|"TPID must be empty or valid FIO address"|
 |Actor not signer|The signer and the `actor` do not match |403| | |Type: invalid_signature|
+|Not owner of marketplace|The signer does not own the marketplace|403| | | |
 
-#### Response
-TBD
-
-##### Example
-TBD
-
-### Set Listing Duration
-
-#### New action: *setmkplstdur*
-This sets the listing duration, in days.
-
-#### Request
-|Parameter|Required|Format|Definition|
-|---|---|---|---|
-|actor|Yes|12 character string|Valid actor of signer (buyer)|
-|marketplace|Yes|1-25 character string|Valid, registered marketplace|
-|listing_dur|Yes|Number less than 365|Duration that the listing will last. There is a check to make sure the domain does not expire within this time frame.|
+#### Response body
+|Parameter|Format|Definition|
+|---|---|---|
+|status|String|OK if successful|
+|new_listing_fee|int|New listing fee amount, in SUFs| 
+|fee_collected|Int|Amount of SUFs collected as fee|
 
 ##### Example
 ```
 {
-    "actor": "2odzomo2v4pe",
-    "marketplace": "somemarketplace"
-    "listing_dur": "120"
+  "status": "OK",
+  "fee_collected": 2000000000
 }
 ```
-#### Processing
-- Request is validated per exception handling
-- Find marketplace
-- Modify listing duration
-
-#### Exception handling
-|Error condition|Trigger|Type|fields:name|fields:value|Error message|
-|---|---|---|---|---|---|
-|Invalid marketplace|Supplied `marketplace` is not found in marketplace config table `mrkplconfig`|400| |"Marketplace not found"|
-|Invalid Listing Duration|`duration` must be positive number less than 365|400| | |"Duration can only be between 1 and 365"|
-|Actor not signer|The signer and the `actor` do not match |403| | |Type: invalid_signature|
-
-#### Response
-TBD
-
-##### Example
-TBD
-
-### Set Listing Warn Time
 
-#### New action: *setmkpwrntm*
-This sets the listing warn time, in days.
+### Set E-Break
+#### Contract: fio.escrow
+#### New fee: set_marketplace_e_break, not bundle eligible
```

Comment:
Does it make sense to have the individual parameter setter instead of just set_config where you set all. Especially if they will not be used often.